style.css

```diff
@@ -0,0 +1,126 @@
+.hero-image {
+    background-image: url('/images/plant-2.jpg');
+    background-repeat: no-repeat;
+    background-size: 40% 100%;
+    background-position: center right;
+    position: relative;
+    min-height: 55rem;
+    margin-bottom: 5rem;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.6);
+
+}
+
+.hero-image-content {
+    width: 35%;
+    position: absolute;
+    top: 30%;
+    left: 10%;
+}
+
+.hero-image-content span {
+    font-size: 3rem;
+    color: var(--light-black);
+}
+
+.hero-image-content h3 {
+    font-size: 6rem;
+    margin-bottom: 1rem;
+    opacity: 0.90;
+}
+
+.shop-now-btn {
+    display: inline-block;
+    padding: 1.2rem 3rem;
+    font-size: 1.7rem;
+    color: var(--white-color);
+    background-color: var(--primary-color);
+    cursor: pointer;
+}
+
+.shop-now-btn:hover {
+    background-color: var(--light-black);
+}
+
+.banner-container {
+    display: flex;
+    gap: 10rem;
+    width: 80%;
+    margin: 0 auto;
+    margin-bottom: 5rem;
```

Comment:
Improvement: you might missed it, redundancy of margin.

Reply:
Thanks for pointing it out, didn't notice.